cpp/inc/bond/core/schema.h

```diff
@@ -202,7 +213,7 @@ class InitSchemaDef
 
         field.id = id;
         field.metadata = metadata;
-        field.type = GetTypeDef<typename remove_maybe<T>::type>();
```

Comment:
Is the `remove_maybe` not needed because `FieldTemplate` already does this?

Reply:
Same reason. The `T` used to be the actual value type (i.e. `FieldT::value_type`) which could be a `maybe<FieldT::field_type>`. While now we are receiving a `value` or ` bonded` of `FieldT::field_type`. Here we do not really care. Otherwise, `FastPathType` would call into an overload that receives the full `FieldT`.